### PR TITLE
UF-337: Rich Text HTML Editor

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

File: uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/htmleditor/HtmlEditorView.css [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+.wysiwyg-clear-right {
+    clear: right;
+}
+
+.wysiwyg-clear-left {
+    clear: left;
+}
+
+
+/* -------- */
```
Comment (ederign) [inline uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/htmleditor/HtmlEditorView.css]: ?

File: uberfire-extensions/uberfire-runtime-plugins/uberfire-runtime-plugins-client/src/main/java/org/uberfire/ext/plugin/client/perspective/editor/layout/editor/popups/EditHTMLView.html [hunk 2]
```diff
@@ -15,15 +13,11 @@
...
-             xmlns:b="urn:import:org.gwtbootstrap3.client.ui">
-
-  <g:HTMLPanel>
-    <b:TextArea size="LARGE" ui:field="textArea"
-                visibleLines="15"/>
-  </g:HTMLPanel>
-
-</ui:UiBinder>
+<div>
+    <div data-field="body" style="height: 500px;">
```
Comment (ederign) [inline uberfire-extensions/uberfire-runtime-plugins/uberfire-runtime-plugins-client/src/main/java/org/uberfire/ext/plugin/client/perspective/editor/layout/editor/popups/EditHTMLView.html]: Please move style to a css file.

**Comment by ederign** [comment]
+1  just some minor comments. 

@paulovmr thanks !

**Comment by paulovmr** [comment]
@ederign Fixing it now, thanks for the review! I'll let you know when it is ready for merge.

**Comment by paulovmr** [comment]
@ederign Done.

**Comment by ederign** [comment]
thanks @paulovmr , waiting for jenkins

**Pull request merged**